src/Core/src/Platform/iOS/SliderExtensions.cs

```diff
@@ -32,10 +32,7 @@ public static void UpdateMinimumTrackColor(this UISlider uiSlider, ISlider slide
 		public static void UpdateMaximumTrackColor(this UISlider uiSlider, ISlider slider)
 		{
 			if (slider.MaximumTrackColor != null)
-			{
-				if (uiSlider.TraitCollection.UserInterfaceIdiom != UIUserInterfaceIdiom.Mac)
-					uiSlider.MaximumTrackTintColor = slider.MaximumTrackColor.ToPlatform();
-			}
+				uiSlider.MaximumTrackTintColor = slider.MaximumTrackColor.ToPlatform();
```

Comment:
Ideally we should have UITests that verify the appearance on iOS and macOS using Mac Idiom. Should we create another testing project with the Mac Idiom set in the Info.plist and with specific tests?. 

Thoughts @PureWeen, @rmarinho?

Reply:
Note that regardless of the Idiom style set in the info.plist, you can change how controls behave by setting the "PreferredBehavioralStyle" on supported controls, so you can mix and match desktop and mobile idioms in the same app.

For a test case, having it be all desktop or all mobile should be fine. A user could mix and match them on their own, but if you're doing that then you're probably a power user who could work through the issues.

Reply:
@jsuarezruiz added a UI test to verify ```MaximumTrackColor``` is set as expected for both ```iOS``` and ```Mac Catalyst```. 

Conscious there's still some work to make that test useful to ```Mac Catalyst``` but that work is potentially broader in scope than this specific issue and are perhaps things to address separately. Notably:

- Supporting use of the ```VerifyScreenshot``` test function for ```Mac Catalyst```
- Establishing approach to testing against ```Mac idiom``` in general

@PureWeen @jsuarezruiz thoughts?

Reply:
Yes @mikeparker104  I think we can create 2 issues to track those 2 issues that some needed. 
- For Screenshot someone just needs to see what its needed or use a 2nd approach
- We already have a test matrixtthat does Windows packages/Unpackaged so I guess we can ase s ame approach to turn the sample with Mac idiom "on".